sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
-    </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
-  </student-page-container>
+  <v-form ref="startApplicationForm">
+    <modal-dialog-base
+      title="Application already in progress"
+      dialogType="warning"
+      :showDialog="showDialog"
+      @dialogClosed="closeDialog"
+    >
+      <template v-slot:content>
+        <v-container>
+          <form>
+            <p>
+              There is already an application selected with this program year.
+              Please continue with that application or cancel it to start a new
+              application.
+            </p>
+          </form>
+        </v-container>
+      </template>
+      <template v-slot:footer>
+        <footer-buttons
+          primaryLabel="Close"
+          :showSecondaryButton="false"
+          @primaryClick="closeDialog"
+        />
+      </template>
+    </modal-dialog-base>
+
+    <student-page-container container-class="container-max-width ma-auto">
+      <template #header>
+        <header-navigator
+          title="Applications"
+          subTitle="Start New Application"
+          :routeLocation="{
+            name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+          }"
+        >
+          <template #buttons>
+            <start-application />
+          </template>
+        </header-navigator>
+      </template>
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for funding
```

Comment:
Apply for **new** funding as per figma. 
![image](https://user-images.githubusercontent.com/54600590/199791995-bfaff843-bdee-452e-923f-57d5d9e3cd8c.png)

Reply:
oh, yes. Thanks!

Reply:
I reverted back to 'Apply for funding' because there was a confusion: there are two place with that screen and they differed but @hellolynn-tbtb fixed.